pkg/features/kube_features.go

```diff
@@ -131,6 +131,12 @@ const (
 	// Enable resource managers to make NUMA aligned decisions
 	TopologyManager featuregate.Feature = "TopologyManager"
 
+	// owner: @cynepco3hahue(alukiano) @cezaryzukowski @k-wiatrzyk
+	// alpha:: v1.20
+
+	// Allows to set containers memory affinity according to NUMA topology
```

Comment:
nit (IMO OK to fix later)
```suggestion
	// Allows setting memory affinity for a container based on NUMA topology
```

Reply:
done